scripts/restore/restore-harbor.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# Check https://compliantkubernetes.io/operator-manual/disaster-recovery/ for instructions
+# Restores latest backup from S3_BUCKET. $1 can be used to specify backup
+# To get a list of available backups use:
+# aws s3 ls ${S3_BUCKET}"/backups" --recursive --endpoint-url=$S3_REGION_ENDPOINT
+set -e
+
+HOSTNAME=localhost
+backup_dir=backups
+
+s3_download() {
+    : "${S3_BUCKET:?Missing S3_BUCKET}"
+    : "${S3_REGION_ENDPOINT:?Missing S3_REGION_ENDPOINT}"
+    if [[ -n "$1" ]]; then
+        backup_key=$1
+    else
+        backup_key=$(aws s3 ls "${S3_BUCKET}/backups" \
+          --recursive \
+          --endpoint-url="${S3_REGION_ENDPOINT}" \
+          | sort | tail -n 1 | awk '{print $4}')
+    fi
+    echo "Downloading backup from s3 bucket ${backup_key}" >&2
+    aws s3 cp "s3://${S3_BUCKET}/${backup_key}" harbor.tgz --endpoint-url="${S3_REGION_ENDPOINT}"
+}
+
+extract_backup(){
+    echo "Extracting backups">&2
+    tar xvf harbor.tgz
```

Comment:
```suggestion
    tar -xvf harbor.tgz -C ${backup_dir}/
```

Or maybe the backup from S3 always have this folder?